install/OWNERS

```diff
@@ -1,5 +1,11 @@
-# No changes should be made here
-# Make all changes to istio-installer
 approvers:
```

Comment:
I think the list is a bit too large here. Can you select from the people who contributed recently and are active - and keep it to 4-5 ?